go/libkb/globals.go

```diff
@@ -312,26 +322,38 @@ func (g *GlobalContext) Logout() error {
 }
 
 func (g *GlobalContext) ConfigureLogging() error {
-	style := g.Env.GetLogFormat()
-	debug := g.Env.GetDebug()
-	logFile := g.Env.GetLogFile()
-	if logFile == "" {
-		g.Log.Configure(style, debug, g.Env.GetDefaultLogFile())
-	} else {
-		g.Log.Configure(style, debug, logFile)
-		g.Log.RotateLogFile()
-	}
-	g.Output = os.Stdout
-	g.VDL.Configure(g.Env.GetVDebugSetting())
+	g.configureLoggingOnce.Do(func() {
+		style := g.Env.GetLogFormat()
+		debug := g.Env.GetDebug()
+		logFile := g.Env.GetLogFile()
+		if logFile == "" {
+			g.Log.Configure(style, debug, g.Env.GetDefaultLogFile())
+		} else {
+			g.Log.Configure(style, debug, logFile)
+			g.Log.RotateLogFile()
+		}
+		g.Output = os.Stdout
+		g.VDL.Configure(g.Env.GetVDebugSetting())
+	})
 	return nil
 }
 
 func (g *GlobalContext) PushShutdownHook(sh ShutdownHook) {
 	g.ShutdownHooks = append(g.ShutdownHooks, sh)
 }
 
-func (g *GlobalContext) ConfigureConfig() error {
-	err := RemoteSettingsRepairman(g)
+func (g *GlobalContext) ConfigureConfig() (err error) {
```

Comment:
I think in `systests` we need this to be called multiple files.

Reply:
Seems to work without it though.

Reply:
Can we let `ConfigureConfig` actually run twice if called twice?